internal/cli/pipeline_inspect.go

```diff
@@ -142,47 +142,76 @@ func (c *PipelineInspectCommand) Run(args []string) int {
 		},
 	}
 
-	s := uint64(c.flagRunSequence)
-	if s > 0 {
-		run, err := c.project.Client().GetPipelineRun(c.Ctx, &pb.GetPipelineRunRequest{
+	var (
+		sha          string
+		msg          string
+		startJobTime string
+		endJobTime   string
+		pipelineRun  *pb.PipelineRun
+	)
+
+	seq := uint64(c.flagRunSequence)
+	if seq > 0 {
+		runResp, err := c.project.Client().GetPipelineRun(c.Ctx, &pb.GetPipelineRunRequest{
 			Pipeline: &pb.Ref_Pipeline{
 				Ref: &pb.Ref_Pipeline_Id{
 					Id: resp.Pipeline.Id,
 				},
 			},
-			Sequence: s,
+			Sequence: seq,
 		})
 		if err != nil {
 			c.ui.Output(clierrors.Humanize(err), terminal.WithErrorStyle())
 			return 1
 		}
+
+		pipelineRun = runResp.PipelineRun
+	} else {
+		pipelineRun = runs.PipelineRuns[len(runs.PipelineRuns)-1]
```

Comment:
Do we need to guard on `len(runs.PipelineRuns)` being `0` here?

Reply:
Good idea! I added a more generic block on this further up in 0495330cb26b59faefb92394882e1f4154a6f27c since most of this code won't make any sense if a pipeline has 0 runs!